cupy/_environment.py

```diff
@@ -330,12 +340,23 @@ def _get_preload_logs():
 def _preload_warning(lib, exc):
     config = get_preload_config()
     if config is not None and lib in config:
-        warnings.warn('''
+        msg = '''
 {lib} library could not be loaded.
 
 Reason: {exc_type} ({exc})
 
 You can install the library by:
+'''
+        if config['packaging'] == 'pip':
+            msg += '''
   $ python -m cupyx.tools.install_library --library {lib} --cuda {cuda}
-'''.format(lib=lib, exc_type=type(exc).__name__, exc=str(exc),
-           cuda=config['cuda']))
+'''
+        elif config['packaging'] == 'conda':
+            msg += '''
+  $ conda install -c conda-forge {lib}
```

Comment:
Question: Is it command enough to pick a correct library version? (`cudatoolkit=x.y` should be specified in the same command line as in the install guide?)

Reply:
So the scenario I expect is like this: Users have successfully installed CuPy with the correct pick of cudatoolkit, but do not install the optional libs. Then at runtime we recommend users to do so if needed. In this case, when doing conda install, conda will pick the right libs that work with the (already installed) cudatoolkit version, so we don't need to add cudatoolkit to the install command explicitly. (Doing so has a small chance to reinstall it if for some reason we bump the build number in the cudatoolkit feedstock, which is no harm but a bit wasting bandwidth and time.)

Reply:
Thanks for clarification! So conda will pick a correct package for  `cupy` + `cudatoolkit` currently installed in the environment.

Reply:
That's right!